samples/matter/lock/prj_thread_wifi_switched.conf

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
+# Enable CHIP
+CONFIG_CHIP=y
+CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32774 == 0x8006 (example lock-app)
+CONFIG_CHIP_DEVICE_PRODUCT_ID=32774
+CONFIG_STD_CPP14=y
+
+# Add support for LEDs and buttons on Nordic development kits
+CONFIG_DK_LIBRARY=y
+
+# Bluetooth Low Energy configuration
+CONFIG_BT_DEVICE_NAME="MatterLock"
+
+# Other settings
+CONFIG_THREAD_NAME=y
+CONFIG_MPU_STACK_GUARD=y
+CONFIG_RESET_ON_FATAL_ERROR=n
```

Comment:
CONFIG_EXPERIMENTAL cannot be set directly. It must be selected by another Kconfig. And given how the experimental-doc script works we would need to add another symbol, such as CHIP_OVER_THREAD_AND_WIFI, to avoid setting experimental flag for the entire nRF53 column.

Reply:
I can do it though :)

Reply:
Done